jetbrains-core/src/software/aws/toolkits/jetbrains/services/codewhisperer/telemetry/CodeWhispererCodeCoverageTracker.kt

```diff
@@ -111,14 +111,17 @@ abstract class CodeWhispererCodeCoverageTracker(
             addAndGetAcceptedTokens(it.endOffset - it.startOffset)
         }
 
-        CodewhispererTelemetry.codePercentage(
-            project = null,
-            acceptedTokensSize,
-            language,
-            percentage,
-            startTime.toString(),
-            totalTokensSize
-        )
+        // percentage == 0 means totalTokens == 0 and users are not editing the document, thus we shouldn't emit telemetry for this
+        percentage?.let { percentage ->
+            CodewhispererTelemetry.codePercentage(
```

Comment:
should this be a passive metric?

Reply:
ack, PR https://github.com/aws/aws-toolkit-common/pull/345